SinaiStCatherine/SinaiEt001.xml

```diff
@@ -0,0 +1,540 @@
+                        <decoNote xml:id="b1">Two wooden boards covered with blind-tooled leather (design of a cross within a frame).</decoNote>
+                        <decoNote xml:id="b2" type="Boards"><term key="textileInlay">Textile inlays</term>.</decoNote>
+                        <decoNote xml:id="b3" type="bindingMaterial">
+                          <material key="wood"/>
+                          <material key="leather"/>
+                          <material key="textile"/>
+                        </decoNote>
+                        <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                     </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                 <origin>
+                   <origDate notBefore="1801" notAfter="1818" evidence="reign"/>
+                 </origin>
+               </history>
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                       <source>
+                         <listBibl type="catalogue">
+                           <bibl>
+                             <ptr target="bm:Kamil1957Sinai"/>
+                             <citedRange unit="page">83-85</citedRange>
+                           </bibl>
+                         </listBibl>
+                       </source>
+                     </recordHist>
+                  </adminInfo>
+
+               </additional>
+
+            </msDesc>
+
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <projectDesc>
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <textClass>
+            <keywords>
+               <term key="OldTestament"/>
+               <term key="Bible"/>
+               <term key="ChristianLiterature"/>
+              <term key="NewTestament"/>
+              <term key="Poetry"/>
+                       </keywords>
+         </textClass>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+           <language ident="grc">Greek</language>
+           <language ident="ar">Arabic</language>
+           <language ident="am">Amharic</language>
+         </langUsage>
+      </profileDesc>
+      <revisionDesc>
+         <change who="DR" when="2022-10-18">Created record from catalogue description with consultation of digitized microfilm</change>
+      </revisionDesc>
+  </teiHeader>
+  <facsimile>
+    <graphic url="https://www.loc.gov/resource/amedmonastery.00279385706-ms/?sp=1&amp;st=image"/>
```

Comment:
nb these images are all served through IIIF, and the manifest is provided
![image](https://user-images.githubusercontent.com/17987728/198119666-1cfd2918-d4a7-4508-8db2-744248834e32.png)
 (https://www.loc.gov/item/00279385706-ms/manifest.json)
the manifest could be given in `idno facs`, I have now enabled it in the code

Reply:
Great, thank you! I had been meaning to ask, but then forgot.